agent_status/scripts/controllers/AgentSummaryController.js

```diff
@@ -173,4 +177,142 @@ mainApp.controller('AgentSummaryController', function ($scope, $state, $timeout,
         });
     };
     $scope.getProfileDetails();
+
+    var setResourceData = function (profile) {
+        if (profile.slotState == 'Reserved') {
+            $scope.ReservedProfile.push(profile);
```

Comment:
Its good if we can use single object to keep all arrays such as $scope.StatusList.ReservedProfile, then code is more organized